src/main/webapp/college-info.html

```diff
@@ -29,8 +29,19 @@ <h1 id="school-name"></h1>
         <p id="race-piechart"></p>
         <p id="gender-piechart"></p>
       </section>
-      <a class="reviews-button" id="reviews-button">Student Reviews</a>
+
+        <h2>Student Reviews</h2>
```

Comment:
our HTML changes are pretty similar, be careful to look out for merge conflicts if I merge before you